### PR TITLE
Added isolated directory for React #22

### LINKED ISSUE #22: Migrate to React
- [ ] Integrate React into the front end development
- [ ] Wherever it is practical, remove jQuery dependencies

### PR DESCRIPTION
**Pull request by prestonmasseyblake**

### CODE REVIEW

File: PantryNodeReact/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# PantryNode
+
+This is the typescript react frontend of Pantry Node
```
Comment (decoles) [inline PantryNodeReact/README.md:3]: Possibly add the steps to build and run?

Comment (prestonmasseyblake) [reply PantryNodeReact/README.md:3]: great idea adding commands to the read me

File: PantryNodeReact/package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "typescript": "^4.9.5",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
```
Comment (AbhinavReddy-Dev) [inline PantryNodeReact/package.json:22]: Please let the testing dependencies remain untouched until we start writing tests for the components.

File: PantryNodeReact/public/manifest.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```
Comment (AbhinavReddy-Dev) [inline PantryNodeReact/public/manifest.json:2]: Need to update this.

File: PantryNodeReact/public/manifest.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  "name": "Create React App Sample",
+  "icons": [
+    {
+      "src": "favicon.ico",
```
Comment (AbhinavReddy-Dev) [inline PantryNodeReact/public/manifest.json:6]: Have a generalized icon replace this.

File: PantryNodeReact/src/App.css [hunk 5]
```diff
@@ -0,0 +1,38 @@
+.App {
```
Comment (AbhinavReddy-Dev) [inline PantryNodeReact/src/App.css:1]: Need to remove this file once we configure Tailwind CSS.

File: PantryNodeReact/src/App.tsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
```
Comment (AbhinavReddy-Dev) [inline PantryNodeReact/src/App.tsx:5]: Need to setup Routing and State Management.

**Review by AbhinavReddy-Dev — APPROVED** [review]
Looks alright to me.

**Comment by AbhinavReddy-Dev** [comment]
I'm not sure if we can close the #22 issue yet, we haven't completely finished the migration. I think we need to decide on the point where we can close the issue. Until then we can create sub-issues related to that main issue, like creating a folder for react, setting up Tailwind CSS, setting up routing, setting up state management etc. Please add your thoughts!

**Review by hardikpatil — APPROVED** [review]

**Pull request merged**